subprojects/docs/README.md

```diff
@@ -148,18 +150,24 @@ To run tests for a single sample, let's say from `samples/java/application`:
 ./gradlew :docs:docsTest --tests "org.gradle.docs.samples.DependencyManagementSnippetsTest.java-application*"
 ```
 
-To run tests for a single snippet:
+#### Code snippets
 
-Let's say you want to run the snippet found at `src/snippets/dependencyManagement/customizingResolution-consistentResolution`.
+Snippets live under `src/snippets`.
 
-then you can run the following command line:
+As an example, you can run tests from [`src/snippets/java/toolchain-task/`](src/snippets/java/toolchain-task) using:
+```
+./gradlew :docs:docsTest --tests "*.snippet-java-toolchain-task_*"
+```
 
+This will run both Kotlin and Groovy tests. You can filter the tests for a specific DSL like this:
 ```
-   ./gradlew :docs:docsTest --tests "*.snippet-dependency-management-customizing-resolution-consistent-resolution*"
+./gradlew :docs:docsTest --tests "*.snippet-java-toolchain-task_kotlin_*"
 ```
 
 which would run both Groovy and Kotlin tests.
 
+#### Testing with configuration cache
+
 It is possible to run samples and snippets with the configuration cache enabled to ensure compatibility.
 To do that set the Gradle property `enableConfigurationCacheForDocsTests=true` in the command line or in the `gradle.properties` file.
```

Comment:
❔ Is this a project property or a system property? So `-P` or `-D`? Or do both work?

Reply:
Seems to be a project property. I tested it, and added a full example to the readme.

https://github.com/gradle/gradle/blob/458c43b66595c0cb5afba1f075ce0656d75fbf71/.teamcity/src/main/kotlin/configurations/DocsTest.kt#L93